docs/pages/ds-audio_L3_Python_TestSpecification.md

```diff
@@ -0,0 +1,143 @@
+# Audio Settings L3 Python Test Specification
+
+## Table of Contents
+
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [Overview](#overview)
+- [References](#references)
+- [Level 3 Python Tests](#level-3-python-test-cases-high-level-overview)
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL`    - Hardware Abstraction Layer
+- `API`    - Application Programming Interface
+- `L2`     - Level 2 Testing
+- `L3`     - Level 3 Testing
+- `DS`     - Device Settings
+- `DUT`    - Device Under Test
+- `NA`     - Not Applicable
+- `RAFT`   - Rapid Automation Framework for Testing
+
+## Overview
+
+This document describes the L3 Test Procedure for the Device Settings Audio module.
+
+## References
+
+- dsAudio HAL Interface - [dsAudio.h](https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/include/dsAudio.h)
+- High Level Test Specification - [ds-audio_High-Level_TestSpecification.md](https://github.com/rdkcentral/rdk-halif-device_settings/blob/main/docs/pages/ds-audio_High-Level_TestSpecification.md)
+- `RAFT` - [`RAFT`](https://github.com/rdkcentral/python_raft/)
+- ut_raft - [ut_raft](https://github.com/rdkcentral/ut-raft)
+
+## Level 3 Python Test Cases High Level Overview
+
+The class diagram below illustrates the flow of dsAudio L3 Python test cases:
+
+```mermaid
+---
+title: dsAudio - Python Class Flow
+---
+classDiagram
+    testControl <|-- ut_raft
+    class ut_raft{
+        class~utMenu~ menuParser
+        class~utUserResponse~ useResponse
+        class~utPlayer~ player
+        class~utConfigurationReader~ configReader
+    }
+    ut_raft <|-- dsAudio
+    dsAudio <|-- L3TestClasses
+
+```
+
+- **testControl**: Test Control Module for running rack Testing. This module configures the `DUT` based on the rack configuration file provided to the test. This class is defined in `RAFT` framework. For more details refer [RAFT](https://github.com/rdkcentral/python_raft/blob/master/README.md)
+- **ut_raft**: Python based testing framework for writing engineering tests. It provides a modular, config driven, low level testing framework. [ut-raft](https://github.com/rdkcentral/ut-raft). It has below sub-classes
+
+|#|Class Name|Description|
```

Comment:
remove this, it's internal to the code, your trying to describe the relationships, and what functions are in there the engineers will require to look.

Reply:
done